src/components/04-table/table--stacked.njk

```diff
@@ -0,0 +1,138 @@
+<div class="width-mobile">
```

Comment:
Might be better to instruct the user to change width of the screen here, so they can see how the stacked tables appear on both wide and narrow scree

Reply:
I added a note in the table caption that says this is a demonstration of the table on mobile screens. I think it's important to demo what's "stacked", even if it seems like it should be obvious, but I'm going to circulate this some more and make sure it's effective.